cswinrt/code_writers.h

```diff
@@ -4925,6 +4936,25 @@ IObjectReference IWinRTObject.NativeObject => _inner;)");
                 w.write(R"(
 global::System.Collections.Concurrent.ConcurrentDictionary<global::System.RuntimeTypeHandle, IObjectReference> IWinRTObject.QueryInterfaceCache { get; } = new();)");
                 }
+
```

Comment:
Couple lines above this, I find myself wondering whether _defaultLazy = new Lazy<%>(() => (%)new IInspectable(((IWinRTObject)this).NativeObject)); should be using _inner or not, what do you think?

Reply:
all references to _inner should be safe, with SetInner initializing them from the most derived ctor, so should be fine

Reply:
I agree with that, I was referring to the use of NativeObject in the default lazy initialization as it comes from IWinRTObject and since all classes implement the interface, I think the one from the child will be used.  The cast after probably makes it the right default interface type, but was wondering if we should just give it _inner if it is now correct.

Reply:
oh, was that another workaround? I missed it. Yes will change that to _inner

Reply:
i am not sure if it was a workaround, it was part of the IDIC pr itself.

Reply:
also the one on line 4621